lib/mirage.ml

```diff
@@ -443,7 +443,9 @@ let archive_conf = impl @@ object
     method name = "archive"
     method module_name = "Tar_mirage.Make_KV_RO"
     method packages =
-      Key.pure [ package ~min:"0.7.0" ~ocamlfind:["tar.mirage"] "tar-format" ]
+      Key.pure [
+        package ~min:"0.7.0" ~ocamlfind:["tar-format.mirage"] "tar-format"
+      ]
```

Comment:
Ok-- I'll update the PR to version constraint `min:"0.8.0"` and to use `tar`/`tar-mirage` appropriately then. Thanks!